coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -65,3 +67,29 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
             file.writelines(ignore_diff.modified)
 
         return file_diff_dict
+
+    def get_ignore_comment(self, origin, language):
```

Comment:
and actually I think *instead of your testcases* a doctest here will provide much more enlightenment and it'll be better readable. Check the deprecate_bear decorator test how it tests logging, I think this'll provide much better readability!